master_server/dummy_vm_server.py

```diff
@@ -0,0 +1,106 @@
+"""
+  This is a dummy VM server.
+  It is built to test master server.
+"""
+import sys
+import os
+import threading
+from datetime import datetime
+import requests
+from flask import Flask
+from flask import request
+import random
+
+
+class MyFlaskApp(Flask):
+  """
+    This class configures the flask app and executes function
+    pre_task before starting flask server.
+  """
+  def run(self, host=None, port=None, debug=None, load_dotenv=True, **options):
+    if not self.debug or os.getenv('WERKZEUG_RUN_MAIN') == 'true':
+      with self.app_context():
+        pre_task()
+    super(MyFlaskApp, self).run(
+        host=host, port=int(sys.argv[1]), debug=debug,
+        load_dotenv=load_dotenv, **options)
+
+app = MyFlaskApp(__name__)
+
+flag = False
```

Comment:
Use caps for constants, http://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations

Reply:
I have made the changes as suggested.